sfdc/errors.bal

```diff
@@ -23,16 +23,21 @@ public type Error error<ErrorDetails>;
 # + headers - Http headers in the error response
 # + body - Response body with extra information  
 public type ErrorDetails record {
-    int? statusCode?; 
+    int? statusCode?;
     map<string[]>? headers?;
-    anydata[]? body?;
+    anydata? body?;
 };
 
 // Error constants
 const string JSON_ACCESSING_ERROR_MSG = "Error occurred while accessing the JSON payload of the response.";
 const string XML_ACCESSING_ERROR_MSG = "Error occurred while accessing the XML payload of the response.";
 const string TEXT_ACCESSING_ERROR_MSG = "Error occurred while accessing the Text payload of the response.";
-const string HTTP_CLIENT_ERROR = "Failed to establish the communication with the upstream server or a data binding failure. Refer error.cause()";
+const string HTTP_CLIENT_ERROR = "Failed to establish the communication with the upstream server or a data binding failure. Refer error.cause() for more details";
 public const string ERR_EXTRACTING_ERROR_MSG = "Error occured while extracting errors from payload.";
 public const string HTTP_ERROR_MSG = "Error occurred while getting the HTTP response.";
 public const string INVALID_CLIENT_CONFIG = "Invalid values provided for client configuration parameters.";
+const STATUS_CODE = "statusCode";
+const HEADERS = "headers";
+const BODY = "body";
+
+
```

Comment:
Shall we remove these unnecessary new lines?

Reply:
Removed. thanks